libmamba/src/specs/match_spec.cpp

```diff
@@ -528,6 +533,47 @@ namespace mamba::specs
             }
         }
 
+        // Handle PEP 440 "Compatible release" specification
+        // See: https://peps.python.org/pep-0440/#compatible-release
+        //
+        // Find a general replacement of the encoding of `~=` with `>=,.*` to be able to parse it
+        // properly.
+        //
+        // For instance:
+        //
+        //     "~=x.y" must be replaced to ">=x.y,x.*" where `x` and `y` are positive integers.
+        //
+        // This solution must handle the case where the version is encoded with `~=` within the
+        // specification for instance:
+        //
+        //                     ">1.8,<2|==1.7,!=1.9,~=1.7.1 py34_0"
+        //
+        // must be replaced with:
+        //
+        //                     ">1.8,<2|==1.7,!=1.9,>=1.7.1,1.7.* py34_0"
+        //
+        while (raw_match_spec_str.find("~=") != std::string::npos)
+        {
+            std::cout << "raw_match_spec_str: " << raw_match_spec_str << std::endl;
```

Comment:
This looks like a leftover for debugging, it should be removed (or use the LOG macros).